src/components/Gamecontent.js

```diff
@@ -39,7 +39,7 @@ class Gamecontent extends Component {
 
   componentWillMount() {
   	if(isEmpty(this.state.gameContent)) {
-  		var gameID = window.location.pathname.substring(1)
+  		var gameID = window.location.pathname.substring(1).replace('games/','')
```

Comment:
This doesn't look right. Why do you need this hard-coding? I don' think it going to fix the issue.

Reply:
I have fixed all.
Please review.